frontend/src/main/components/Driver/DriverAvailabilityForm.js

```diff
@@ -11,7 +11,7 @@ function DriverAvailabilityForm({ initialContents, submitAction, buttonLabel = "
     } = useForm(
         { defaultValues: initialContents || {}, }
     );
-    // Stryker restore all
+    // Stryker enable all
```

Comment:
```suggestion
    // Stryker restore all
```

It turns out that `    // Stryker enable all` isn't a thing.  We need to change all occurences of `// Stryker enable all` to `// Stryker restore all` as find them.